openvm/src/lib.rs

```diff
@@ -1279,15 +1283,21 @@ mod tests {
         );
     }
 
-    // #[test]
-    // #[ignore = "Too much RAM"]
-    // // TODO: This test currently panics because the kzg params are not set up correctly. Fix this.
-    // #[should_panic = "No such file or directory"]
-    // fn keccak_prove_recursion() {
-    //     let mut stdin = StdIn::default();
-    //     stdin.write(&GUEST_KECCAK_ITER);
-    //     prove_recursion(GUEST_KECCAK, GUEST_KECCAK_APC, GUEST_KECCAK_SKIP, stdin);
-    // }
+    #[test]
+    #[ignore = "Too much RAM"]
+    fn keccak_prove_recursion() {
```

Comment:
There's no need for KZG params anymore because we only run STARK recursion, so this runs now on our server.

Reply:
Yes but there were reports in your PR doing degree stuff from a few months ago I think that recursion was an issue with the degree change, so I think it's good to test this together (done manually and automatically with nightly tests)